wallet/src/components/SmartWalletConnection.tsx

```diff
@@ -1,8 +1,9 @@
 import { makeFollower, makeLeader } from '@agoric/casting';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
why is this line necessary? the dep is in package.json

Reply:
Good catch, I think a remnant from when I was `yarn link`ing